src/components/molecules/NavbarRadio/NavbarRadio.scss

```diff
@@ -0,0 +1,10 @@
+.radio-trigger {
+  position: relative;
+  padding: 0.5em;
+}
+
+.radio-wrapper {
+  position: absolute;
+  top: 100%;
+  right: 0;
+}
```

Comment:
Add back the missing styles here using the BEM modifier pattern, rather than as inline styles in the component:

```typescript
export const RadioWrapper = styled.div<RadioWrapperProps>`
  position: absolute;
  top: 100%;
  right: 0;
  visibility: ${({ showRadioPopover }) =>
    showRadioPopover ? "visible" : "hidden"};
  user-select: ${({ showRadioPopover }) =>
    showRadioPopover ? "auto" : "none"};
`;
```

Reply:
I thought you wanted to get rid of `styled-components` ?

Reply:
I do, but this is the copy/pasted code of the styles as they existed before they were split apart into scss and inline styles, so that you can reference it without having to go looking.